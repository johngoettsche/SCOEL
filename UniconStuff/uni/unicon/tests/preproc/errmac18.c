/* test error message: redefined macros must have identical bodies */

#define m 1
#define m 2
