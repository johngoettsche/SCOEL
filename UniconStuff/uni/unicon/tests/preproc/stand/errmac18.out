pp: File errmac18.c; Line 4: invalid redefinition of macro m
